sirepo/raydata/scan_monitor.py

```diff
@@ -361,11 +361,13 @@ async def _process_analysis_queue():
                 ) as l:
                     try:
                         for n in d.get_notebooks():
+                            d.render_papermill_script(
+                                input_f=n.input_f,
+                                output_f=n.output_f,
+                            )
                             p = await asyncio.create_subprocess_exec(
-                                "papermill",
-                                str(n.input_f),
-                                str(n.output_f),
-                                *d.get_papermill_args(),
+                                "bash",
+                                d.get_papermill_script_path(),
```

Comment:
As I noted above, `render_papermill_script` should return the path and that should be used here. You can just call to `render_papermill_script` here so there is no need for a temporary.

Reply:
7407ed0e7849051a42a067015819229cd53c406b